MoreLinq.Test/WindowTest.cs

```diff
@@ -1,3 +1,6 @@
+using System.Collections.Generic;
```

Comment:
Move these under namespace declaration, together with other imports in the file.